.github/workflows/dev.yml

```diff
@@ -43,10 +43,16 @@ jobs:
           submodules: recursive
 
       - name: Build eBL API
-        run: cd DCSA-EBL && mvn package -Ddcsa.events.version=$DCSA_EVENT_CORE_PROJECT_VERSION -Ddcsa.event.tag=dev-ebl -Ddcsa.artifacttype=-SNAPSHOT #Need to skip the tests as they depend on a running DB
+        run: cd DCSA-EBL && mvn -B package -Ddcsa.events.version=$DCSA_EVENT_CORE_PROJECT_VERSION -Ddcsa.event.tag=dev-ebl -Ddcsa.artifacttype=-SNAPSHOT #Need to skip the tests as they depend on a running DB
 
-      - name: Build and run the microservices
-        run: cd DCSA-EBL && docker-compose -f docker-compose.yml up -d -V
+      - name: Start the database
+        run: cd DCSA-EBL && docker-compose -f docker-compose.yml up -d -V dcsa-test-db
+
+      - name: Load test data into the database
+        run: sleep 3 && docker exec -i dcsa_db sh -c "cat /testdata.d/* | psql -h localhost -U postgres postgres"
+
```

Comment:
Maybe it is time we stop doing this outside our docker image and instead have it load test data on init (possibly conditionally via an ENV var).  I am getting tired of all the work arounds and issues this causes for our CI.

I would be in favor of overturning our previous decision that the image should only contain non-test data.

Reply:
I guess we need to have a discussion in the team about how to move forward so we are aligned.